src/facades/book.js

```diff
@@ -2,6 +2,16 @@
 
 let books = require('../../db.json')
 
+function getAuthors () {
```

Comment:
Since this function returns _Authors_ I would put this function in its own module under `/facades/author.js`. That way we can keep our modules small and according to SRP (Single Responsibility Principle), they would be responsible only for one thing (i.e. books or authors accordingly).